packages/@aws-cdk/aws-ec2-alpha/lib/subnet-v2.ts

```diff
@@ -145,19 +153,16 @@ export class SubnetV2 extends Resource implements ISubnetV2 {
    */
   public readonly ipv6CidrBlock?: string;
 
-  /**
-   * The route table for this subnet
-   */
-  public readonly routeTable: IRouteTable;
-
   /**
    * The type of subnet (public or private) that this subnet represents.
    * @attribute SubnetType
    */
-  public readonly subnetType: SubnetType;
+  public readonly subnetType?: SubnetType;
```

Comment:
This is not optional under props, so user still needs to defined it but it is marked as optional under the new interface `SubnetV2` because if not it will cause issue with the existing property in [`SubnetSelection` for subnets ](https://github.com/aws/aws-cdk/blob/main/packages/aws-cdk-lib/aws-ec2/lib/vpc.ts#L363) which we are leveraging in some of the places [like here.](https://github.com/aws/aws-cdk/blob/main/packages/%40aws-cdk/aws-ec2-alpha/lib/subnet-v2.ts#L351)